shell/components/formatter/ClusterLink.vue

```diff
@@ -43,10 +44,10 @@ export default {
             formattedTooltip.push(`<p>${ [c.type] } (${ c.status })</p>`);
           });
 
-        return formattedTooltip.toString().replaceAll(',', '');
+        return formattedTooltip.toString().replace(/,/g, '');
```

Comment:
What is the reason for this change, is there an issue with `replaceAll`?

Reply:
that's because `replaceAll()` is Node 16^ only but the GH actions are running the unit tests in Node 14

Reply:
looks like we have a polyfill for replaceAll but it's imported in the app's initialization. I can import it in the test itself to fix the issue and revert the change above